.github/workflows/druid-test-image.yml

```diff
@@ -0,0 +1,33 @@
+name: Build Druid image for tests
+
+on:
+  pull_request:
+    paths:
+      - "drivers/druid/Dockerfile"
```

Comment:
do we need it only for PRs?

Reply:
Yes, it should be rebuilt on change, building on merging doesn't matter.